readthedocs/settings/base.py

```diff
@@ -269,6 +269,27 @@ def USE_PROMOS(self):  # noqa
     # Docker
     DOCKER_ENABLE = False
     DOCKER_IMAGE = 'readthedocs/build:2.0'
```

Comment:
We are losing the previous behavior of this setting in the config module. Previously it was the result of `'{}:{}'.format(DOCKER_DEFAULT_IMAGE, DOCKER_DEFAULT_VERSION)`